features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml

```diff
@@ -1,9 +1,10 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
 <!--
-~ Copyright (c) 2011, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+~ Copyright (c) 2022, WSO2 LLC. (http://www.wso2.com).
```

Comment:
We should not update the year

Reply:
fixed with 3d40db15acfe5463579d07ae8c1dd523be4ce63f